lib/src/particle/particle_callbacks.dart

```diff
@@ -78,7 +76,7 @@ class UpdateBodyContactsCallback implements QueryCallback {
             aabbLowerBoundY <= ap.y &&
             ap.y <= aabbUpperBoundY) {
           double d;
-          final n = _tempVec;
+          final n = Vector2.zero();
```

Comment:
This is one of these ugly optimizations, I don't know how often `reportFixture` will be called, but it would probably be good to keep it.